pkg/operator/configobservation/cloudprovider/observe_cloudprovider.go

```diff
@@ -76,6 +76,9 @@ func ObserveCloudProviderNames(genericListers configobserver.Listers, recorder e
 	case platform["libvirt"] != nil:
 		// this means we are using libvirt
 		return observedConfig, errs
+	case platform["none"] != nil:
```

Comment:
Does the `none` need to be specifies explicitly? Just asking what is the difference when the cloud provider is not specified and when it is set to `none`.

Reply:
`none` is the platform name in the installer, not cloudprovider name